File: Fittify.Web.ApiModels/Sport/Get/WorkoutHistoryForGet.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using Fittify.Web.Common;

namespace Fittify.Web.ApiModels.Sport.Get
{
    public class WorkoutHistoryForGet : UniqueIdentifier<int>
    {
        public DateTime? DateTimeStart { get; set; }
        public DateTime? DateTimeEnd { get; set; }
        
        public int WorkoutId { get; set; }
        public string WorkoutName { get; set; }

        public virtual string RangeOfExerciseHistoryIds { get; set; }
    }
}

File: Fittify.Angular/src/app/apimodels/get/WorkoutHistoryForGet.ts
import {UniqueIdentifier} from "../../common/UniqueIdentifier";

export class WorkoutHistoryForGet extends UniqueIdentifier<number>
{
  public dateTimeStart: Date;
  public dateTimeEnd: Date;
  public workoutId: number;
  public workoutName: string;
  public exerciseHistoryIds: number[];
}
